### ISSUE TITLE
Improve copy on profile page

### ISSUE THREAD

**Issue by matt-west** [Automattic/wp-calypso#100247]
## Where

/me

## Problem

The profile page (/me) contains a lot of copy that could likely be reduced to simplify this page:

* Lengthy explanation of Gravatar
* Notes identifying which fields are public
* Public profile link feels misplaced in the middle of the fields

![Image](https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/9d039cf6-fc7f-412c-bec3-9167dd2e7766/c62592e7-71a9-4127-9927-09f269c4e8df?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi85ZDAzOWNmNi1mYzdmLTQxMmMtYmVjMy05MTY3ZGQyZTc3NjYvYzYyNTkyZTctNzFhOS00MTI3LTk5MjctMDlmMjY5YzRlOGRmIiwiaWF0IjoxNzQ2NjI2MDc3LCJleHAiOjMzMzE3MTg2MDc3fQ.GE3wbQ29XJol8z4hmWLWGSRUdECm_buD7Nv49I2pwO8)

## Solution

- [X] Identify alternative ways to signal public fields
- [X] Simplify Gravatar copy
- [X] Explore alternative ways of displaying the public profile link

**Comment by jboland88** [comment]
@matt-west  - this one needs some design input to define how to better present the profile page. Could you help define the improvements that should be made?

**Comment by matt-west** [comment]
We’ve been [working on a design for this](https://www.figma.com/design/QOBjujZah0UlGDDdLKZhzi/WordPress.com-Hosting-Dashboard?node-id=2140-208336&t=CAysPymzT7d8iAGr-4) as part of the dashboards work, but it’s not quite finished yet. I’ll assign this issue to me and follow up once we have a design that can be retrofitted to the current dashboard.

**Comment by crisbusquets** [comment]
I can help with this. @josh.boland I'll get back to you asap.

**Comment by matt-west** [comment]
@crisbusquets Are you still planning to look at this? I can work on it this week if not.

**Comment by matt-west** [comment]
I’ve opened a [PR](https://github.com/Automattic/wp-calypso/pull/103464) with some changes to clean up this screen. We have some ideas for wider improvements to the layout of forms, but those will be tackled as part of the hosting dashboard redesign.
